dietpi/dietpi-ddns

```diff
@@ -249,7 +264,7 @@ Menu_Provider()
 	G_WHIP_DEFAULT_ITEM=${PROVIDER:-DuckDNS}
 
 	# Custom provider selected
-	if [[ $PROVIDER && $PROVIDER != 'DuckDNS' && $PROVIDER != 'No-IP' && $PROVIDER != 'Dynu' ]]
+	if [[ $PROVIDER && $PROVIDER != 'DuckDNS' && $PROVIDER != 'No-IP' && $PROVIDER != 'Dynu' && $PROVIDER != 'FreeDNS' && $PROVIDER != 'OVH' ]]
```

Comment:
Would it not be easier to say:
```sh
if [[ $PROVIDER && $PROVIDER == 'Custom' ]]
```

Reply:
Yes, you're absolutely right. I think when I started to add this logic, `PROVIDER='Custom'` was not yet set, but added later as a necessity. As it's late and this is a last minute addition, I didn't want to change the logic to avoid errors 😉. With next version (when we add further providers) we need to change this, including a nicer way to check whether username, password or domains need to be input, based on provider. I though about an associated array, which holds that information, including whether the call the password "Password" or "Token" 😄.

Reply:
Ah lol, now I confused it with DietPi-VPN, I think. Here, `PROVIDER='Custom'` is still never set, but the variable holds the update URL instead, when a custom provider is used. But we might want to change that.